account_payment_term_partner_holiday/models/res_partner_holiday.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+
+
+class ResPartnerHoliday(models.Model):
+    _name = "res.partner.holiday"
+    _description = "Partner Holidays"
+
+    @api.model
+    def _get_days(self):
```

Comment:
Follow guidelines for method name (for both):
```suggestion
    def _selection_days(self):
```